pkg/coscheduling/README.md

```diff
@@ -0,0 +1,116 @@
+# Overview
+
+This folder holds the coscheduling plugin implementations based on [Lightweight coscheduling based on back-to-back queue 
+sorting](https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/kep/20200116-lightweight-coscheduling-based-on-back-to-back-queue-sorting.md).
+
+## Maturity Level
+
+<!-- Check one of the values: Sample, Alpha, Beta, GA -->
+
+- [ ] 💡 Sample (for demonstrating and inspiring purpose)
+- [x] 👶 Alpha (used in companies for pilot projects)
+- [ ] 👦 Beta (used in companies and developed actively)
+- [ ] 👨 Stable (used in companies for production workloads)
+
+## Tutorial
+### PodGroup
+We use a special label named pod-group.scheduling.sigs.k8s.io/name to define a PodGroup. Pods that set this label and use the same value belong to the same PodGroup. 
+```
+labels:
+     pod-group.scheduling.sigs.k8s.io/name: nginx
+     pod-group.scheduling.sigs.k8s.io/min-available: "2"
+```
+We will calculate the sum of the Running pods and the Waiting pods (assumed but not bind) in scheduler, if the sum is greater than or equal to the minAvailable, the Waiting pods
+will be created.
+
+Pods in the same PodGroup with different priorities might lead to unintended behavior, so need to ensure Pods in the same PodGroup with the same priority.
+
+### Expectation
+1. If 2 PodGroups with different priorities come in, the PodGroup with high priority has higher precedence.
+2. If 2 PodGroups with same priority come in when there are limited resources, the PodGroup created first one has higher precedence.
+
+### Config
+1. queueSort、permit and unreserve must be enabled in coscheduling.
+2. preFilter is user selectable according to the actual situation of users.
+```
```

Comment:
I add description of the configuration here to tell user can disable the preFilter. @alculquicondor

Reply:
I think you should clarify why a user might want to use it or not.

Reply:
@alculquicondor  Done

Reply:
I kind of disagree with the statement. It depends more on the number of podgroups, rather on the size of a podgroup.